tests/utils/io/test_mutations_report.py

```diff
@@ -603,6 +603,40 @@ def test_filtered_mutation_combine_columns(self):
             self.assertEqual(result[11].rstrip(), "sample1	NC_000002.11	29445282	29445282	G	A	4-other	520	284	3	yes	ok	ALK;-:-	protein_coding	splice_region_variant&intron_variant	vardict	-")  # noqa
             self.assertEqual(result[12].rstrip(), "sample1	NC_000016.11	81954789	81954789	C	G	4-other	1144	442	349	yes	ok	PLCG2;-:-	protein_coding	intron_variant	vardict,mutect2	-")  # noqa
 
+    def test_filtered_mutation_select_one_columns(self):
+        from hydra_genetics.utils.io.hotspot_report import generate_hotspot_report
+        levels = [(300, "ok", "yes"), (30, "low", "yes"), (0, "low", "not analyzable")]
```

Comment:
300 => 200

Reply:
let's add it to our actual pipeline instead of changing the tests